### DISCUSSION TITLE
Add an option to bring the 21996 action center

### DISCUSSION THREAD

**Discussion by lixkote** [valinet/ExplorerPatcher#1110] [Ideas]
You can get it working by replacing the files, but its very unstable.
You can also add an option to make it rouneded like with the Windows 10 Start Menu.
![image](https://user-images.githubusercontent.com/95425619/226119347-2989246a-078c-445f-82df-defd54621f24.png)

**Comment by lixkote** [comment]
@valinet Aside that, you could add an option for a normal Windows 10 Action Center.

**Comment by lixkote** [comment]
@valinet 
Upon further investigation, I have drawn some conclusions; 
**21996/22000.1 action center situation;**
I had an access to a prototype version of 10SM by **@bbmaster123** https://github.com/bbmaster123(I won't disclose it here tho), and that version just replaces the files in `C:\Windows\SystemResources` and` C:\Windows\SystemApps` with the ones from 22000.1, thus making the action center work in modern(later)22000 and early 22621. BUT are some caveats like;
- The Action Center has the same bug as Windows 10 Start Menu, i mean the bug that introduced a 1-2 second delay when the menu was dismissed,
- In some situations, the action center just refuses to open, i managed to capture the exception in WinDbg(See attached 21996accrash.txt)
- on 22621, some toggles are broken.

**Windows 10 action center situation;**
That action center can be fully enabled even in modern 22621 by enabling
```
[HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Control Center]
"UseLiteLayout"=dword:00000001
```
The caveats there are;
- For some reason the action center uses the light theme, regardless of system settings
- The same 1-2 second delay on dismiss bug as with the 21996 one.

For the roundening of the 22000.1/21996 action center, you can use the same method as with the Windows 10 Start Menu rounded corners, see 
https://github.com/valinet/ExplorerPatcher/blob/8e9403cfb53fa10954d1d8e13c9d8988e180f704/ExplorerPatcher/lvt.c
https://github.com/valinet/ExplorerPatcher/blob/7c4567ac799ceaed8fbb582d1e232dfacfa3c354/ExplorerPatcher/lvt.h
Using that, you can add 10,10,10,10 padding on the "RootGrid" Grid Control and add add CornerRadius on the same control. You could also make the acrylic less saturated by setting acrylic to these values:
TintOpacity to 0.5
TintLuminosityOpacity to 0.96
You can also use XamlDiagnostics, which i mentioned earlier, for better understanding see;
https://github.com/Lixkote/10StartModenizer

I would be really thankful if you could try to make them work again, as for many users the action center is a very crucial
[21996accrash.txt](https://github.com/valinet/ExplorerPatcher/files/11044359/21996accrash.txt)
 part of windows experience. 

Regards, lixkote.

**Reply by lixkote** [reply]
Update: i found what makes the 21996 action center crash sometimes... Its the Windows Security notification, which tries to display like in a normal 22000.318 action center, thus breaking the xaml, and making the action center crash.

**Reply by fromhgbwithluv** [reply]
@Lixkote Can you help me to add Windows 10 Layout (Only Network, Battery and Control Center) to Windows 11

**Comment by XuanChen1234** [comment]
You can ask from @Amrsatrio

**Comment by 10aaron10** [comment]
como puedo hacerlo en windows 11,por q me gusta mas el centro de actividades de win10,porfa,responder,quiero usarlo.

**Reply by 10aaron10** [reply]
osea tener el centro de actividades de windows 10 en windows 11

**Comment by SoftwareExplorer6600** [comment]
@lixkote Can you help me with implementing this, even if its just an executable I would want to at least test this
